data/data-pipeline/data_pipeline/tests/sources/national_risk_index/data/output.csv

```diff
@@ -1,56 +1,56 @@
-15007040604,13.8830970216,0.0000023137,0.0000977117,0.0001482071,True
-15009030303,26.4505931648,0.0000000878,0.0000048854,0.0000069565,True
-15009030403,18.4437334890,0.0000000310,0.0000103042,0.0000013105,True
-15009030404,21.7984897602,0.0000000310,0.0000104610,0.0000013088,True
-15001021601,46.2888122040,0.0000022210,0.0001000049,0.0004101879,True
-15001021013,31.7870290791,0.0000000365,0.0000291145,0.0000015088,True
-15001021702,43.0017491363,0.0000038333,0.0006472757,0.0007289681,True
-15001021704,40.5478070100,0.0000000675,0.0000000000,0.0000124075,False
-15001021902,24.1028365867,0.0000012633,0.0003623064,0.0002255654,True
-15003007502,13.4756112773,0.0000000123,0.0000000000,0.0000029166,False
-15001020202,25.4228346534,0.0000002378,0.0000290487,0.0000415197,True
-15001020300,36.0264444526,0.0000009915,0.0000123128,0.0003168032,True
-15003009507,13.7534872042,0.0000000122,0.0000000000,0.0000018990,False
-15009031502,24.1981968780,0.0000006427,0.0000676868,0.0000108687,True
-15009031700,20.0761939648,0.0000010934,0.0049143834,0.0000148370,True
-15001021202,50.9009757018,0.0000000383,0.0000301617,0.0000017093,True
-15001021300,40.9441045600,0.0000039558,0.0000396241,0.0005806917,True
-15003010308,14.8377632042,0.0000000122,0.0000000000,0.0000020149,False
-15001021504,38.7614777296,0.0000001803,0.0000000000,0.0001167168,False
-15001021604,41.0047129501,0.0000009324,0.0000000000,0.0002774903,False
-15001022000,19.5680692358,0.0000000358,0.0000290511,0.0000014444,True
-15001022102,26.8749889144,0.0000000358,0.0000290467,0.0000014219,True
-15001021509,39.6085028569,0.0000003842,0.0000351301,0.0001122053,True
-15003010100,20.1899418692,0.0000000649,0.0009782148,0.0000759598,True
-15003010202,22.0650824666,0.0000002726,0.0017126977,0.0002904509,True
-15003000114,13.0024416326,0.0000000593,0.0000000000,0.0000559929,False
-15003000301,15.5029761891,0.0000000543,0.0000000000,0.0000439802,False
-15003000401,14.3094432099,0.0000000175,0.0000000000,0.0000089358,False
-15003000500,21.7355738392,0.0000001600,0.0000000000,0.0001780227,False
-15003001100,12.9865448527,0.0000000123,0.0000000000,0.0000021143,False
-15003001400,12.3585076356,0.0000000124,0.0000000000,0.0000023264,False
-15003001901,27.1286782082,0.0000003239,0.0000000000,0.0003416043,False
-15003001904,21.1782783212,0.0000003731,0.0000000000,0.0004239595,False
-15003002202,16.8452811998,0.0000003007,0.0000000000,0.0003313697,False
-15003002600,21.1208678596,0.0000000125,0.0000000000,0.0000024799,False
-15003002800,14.0640371335,0.0000000147,0.0000000000,0.0000053072,False
-15003003000,15.9122177661,0.0000000173,0.0000000000,0.0000088501,False
-15003003101,16.7794100011,0.0000000378,0.0000000000,0.0000269712,False
-15003003407,12.1430194003,0.0000000124,0.0000000000,0.0000023264,False
-15003003501,15.9378488267,0.0000000124,0.0000000000,0.0000023264,False
-15003004600,15.6406606485,0.0000000123,0.0000004599,0.0000021180,True
+06001020100,18.6199401875,3.5067e-06,0.0031812618,6.6152e-05,True
```

Comment:
i don't think the output should use scientific notation, right? we've gone out of our way in the past to make sure they write them out with full digits.